dvc/config.py

```diff
@@ -149,6 +149,8 @@ class RelPath(str):
                     "profile": str,
                     "credentialpath": str,
                     "endpointurl": str,
+                    Optional("s3_key_id", default=""): str,
+                    Optional("s3_key_secret", default=""): str,
```

Comment:
These are optional by default, you don't need to set them
```suggestion
                    "s3_key_id": str,
                    "s3_key_secret": str,
```

Reply:
done